descqa/ColorDistribution.py

```diff
@@ -33,7 +34,16 @@
 color_transformation['sdss2cfht']['r'] = 'r - 0.024 * (g - r)'
 color_transformation['sdss2cfht']['i'] = 'i - 0.085 * (r - i)'
 color_transformation['sdss2cfht']['z'] = 'z + 0.074 * (i - z)'
-
+color_transformation['lsst2cfht']['u'] = 'u'
```

Comment:
Are Lines 37 to 46 really needed?  If no transformation is needed, it should not appear here I think...

Reply:
Oops yes, that was left in by mistake.